javascript-create-module/template/prettier.config.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @type {import("prettier").Config}
+ * @see https://prettier.io/docs/en/configuration.html
+ */
+export default {
+  quoteProps: "consistent",
+  printWidth: 100,
+  plugins: ["prettier-plugin-jsdoc", "prettier-plugin-packagejson"],
```

Comment:
The two extensions are probably not necessary for user code, let's merge and rethink that later